NebulaNetwork/PacketProcessors/Session/SyncCompleteProcessor.cs

```diff
@@ -16,7 +16,7 @@ public class SyncCompleteProcessor : PacketProcessor<SyncComplete>
 
         public SyncCompleteProcessor()
         {
-            playerManager = MultiplayerHostSession.Instance?.PlayerManager;
```

Comment:
Isn't this doing the same as it was previously?

Reply:
VS warns against using null propagation on unity objects as they can be invalid without being null

Reply:
https://blog.unity.com/technology/custom-operator-should-we-keep-it

Reply:
Yeah, `?.` on anything defined in unity is dangerous.